frontend/src/app/components/StreamlitDialog/StreamlitDialog.tsx

```diff
@@ -155,17 +155,12 @@ function aboutDialog(props: AboutProps): ReactElement {
             />
           </StyledAboutInfo>
         </ModalBody>
-        <ModalFooter>
-          <ModalButton kind={BaseButtonKind.SECONDARY} onClick={props.onClose}>
-            Close
-          </ModalButton>
-        </ModalFooter>
```

Comment:
Just to confirm for the three cases in this file - we are just removing the extra `Close` button at the bottom of the modal, because we already have an X closure button at the top right of the modal?

Reply:
Exactly! Because you can already close the modal by:
1. Clicking on the `X` button;
2. Clicking outside the modal;
3. Hitting the `Esc` key.